src/main/jte/bookings/calendar.jte

```diff
@@ -8,6 +8,16 @@
 @template.layout(page = new PageSpec("Index", "Soli"), context = context, layout = layout, content = @`
     <script>
         document.addEventListener('DOMContentLoaded', function () {
+            fetch('/api/${layout.getRoom().getId()}/opening-hours') // This part is probably wrong, but i have no idea what the correct way is
+                // The way it gets things like hours.dayOfWeek is probably wrong
+                .then(response => response.json())
+                .then(openingHours => {
+                    const businessHours = openingHours.map(hours => ({
+                        daysOfWeek: [hours.dayOfWeek], // Assuming dayOfWeek is 0 (Sunday) to 6 (Saturday)
+                        startTime: hours.openingTime,
+                        endTime: hours.closingTime
+                    }));
+
```

Comment:
A resource [JSON feed](https://fullcalendar.io/docs/resources-json-feed) containing the [business hours](https://fullcalendar.io/docs/businessHours-per-resource) seems cleaner.